src/app/pages/auth/containers/enrolment-page.component.ts

```diff
@@ -227,10 +227,27 @@ export class EnrolmentPageComponent {
   doAfterAuthentication() {
     this.configService
       .fetchConfigState(true)
+      .catch(e => this.showConfigError())
       .then(() => this.firebaseAnalytics.logEvent('sign_up', {}))
       .then(() => this.next())
   }
 
+  showConfigError() {
+    const buttons = [
+      {
+        text: this.translate.transform(LocKeys.BTN_OKAY.toString()),
+        handler: () => {
+          this.doAfterAuthentication()
+        }
+      }
+    ]
```

Comment:
Should we add a cancel button too? Otherwise this may go into look if there is always error in the doAfterAuthentication() function.